packages/lesswrong/components/recommendations/RecommendationsAlgorithmPicker.tsx

```diff
@@ -39,34 +39,34 @@ const recommendationAlgorithms = [
   }
 ];
 
-function getDefaultSettings(configName) {
-  if (configName in slotSpecificRecommendationSettingDefaults) {
-    return deepmerge(defaultAlgorithmSettings, slotSpecificRecommendationSettingDefaults[configName]);
-  } else {
-    return defaultAlgorithmSettings;
-  }
-}
-
 export function getRecommendationSettings({settings, currentUser, configName})
 {
   if (settings)
    return settings;
 
   if (currentUser?.recommendationSettings && configName in currentUser.recommendationSettings) {
-    return deepmerge(getDefaultSettings(configName), currentUser.recommendationSettings[configName]||{});
+    return deepmerge(defaultAlgorithmSettings, currentUser.recommendationSettings[configName]||{});
   } else {
-    return getDefaultSettings(configName);
+    return defaultAlgorithmSettings;
   }
 }
 
+// TODO: Soon to be removed when Community becomes a tag. At that point, we'll
+// move this into forumLanguage
 const forumIncludeExtra = {
   LessWrong: {humanName: 'Personal Blogposts', machineName: 'includePersonal'},
   AlignmentForum: {humanName: 'Personal Blogposts', machineName: 'includePersonal'},
   EAForum: {humanName: 'Community', machineName: 'includeMeta'},
 }
-
 const includeExtra = forumIncludeExtra[forumTypeSetting.get()]
 
+const forumArchivesName = {
+  LessWrong: 'Archive Recommendations',
+  AlignmentForum: 'Archive Recommendations',
+  EAForum: 'Forum Favorites',
+}
+const archiveName = forumArchivesName[forumTypeSetting.get()]
```

Comment:
As above maybe abstract this into a function that can be used elsewhere

Reply:
Good suggestion, thanks. Doing it for a variable since forumTypeSetting is known at initialization.